### ISSUE TITLE
How to calculate EOM-CCSD(T)

### ISSUE THREAD

**Issue by okunoyukihiro2** [pyscf/pyscf#2257]
Dear PYSCF Users. 

I want to calculate excitation state by EOM-CCSD with (T) correction. 

For ground state, we set 

from pyscf import gto,scf,cc
mol = gto.M(atom='Be  0.0  0.0  0.0; H  0.0  0.0  1.33115; H 0.0 0.0 -1.33115;',basis='sto-3g',symmetry=False,verbose=0)
mf = scf.RHF(mol).run()
ccsd=cc.CCSD(mf)
ecorr,t1,t2 = ccsd.kernel()
et = ccsd.ccsd_t()
print('ground state energy =', ccsd.etot + et)

e,v = ccsd.eeccsd(nroots=10)
#eom = cc.ccsd_eom.EOM(ccsd)                                                                                                        

how to tread energy level of excitation energy by EOM-CCSD(T) ? 

Sincerely,

**Comment by tberkel** [comment]
For IP and EA only (*not* EE), the EOM-(T)* method of [Matthews and Stanton](https://doi.org/10.1063/1.4962910) was implemented, but to be honest, I don't know how extensively it has been tested. It can be used as follows
```
eom = cc.eom_rccsd.EOMIP(ccsd)
e, r = eom.kernel(nroots=3)
e *= 27.21139
print("EIP (eV) =", *e)

eom_t = cc.eom_rccsd.EOMIP_Ta(ccsd)
estar = eom_t.ipccsd_star(nroots=3)
estar = np.array(estar) * 27.21139
print("EIP* (eV) =", *estar)
```

**Comment by okunoyukihiro2** [comment]
Dear 　therkel. 

Thank you for your response. 
I found it both helpful and informative.

I have a simple question. Why is there no EOM-EE implementation that does not change the number of electrons　from the 
reference state?　
Is there a difference in theoretical difficulty?

Sincerely.

**Comment by tberkel** [comment]
No, there is no theoretical difficulty, it's just that no one has yet implemented it in PySCF.

**Comment by verena-neufeld** [comment]
Thanks @okunoyukihiro2 for asking. This might be a `feature request`.
